File: Npc/11001495.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11001495: Startz
/// </summary>
public class _11001495 : NpcScript {
    protected override int First() {
        return 30;
    }

    // Select 0:
    // $script:0118150907005816$
    // - Do you need more food?
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (30, 0):
                // $script:0118150907005819$
                // - I've got a bad feeling... like something bad's about to happen.
                return -1;
        }
        
        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (30, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11001495.py
""" 11001495: Startz """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 30

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0118150907005816$
        # - Do you need more food?
        return None # TODO

    def __30(self, pick: int) -> int:
        # $script:0118150907005819$
        # - I've got a bad feeling... like something bad's about to happen.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (30, 0):
            return Option.CLOSE
        return Option.NONE
